.github/workflows/CompatHelper.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         julia-version:
-          - '1.6'
+          - '1'
```

Comment:
Just to note that this is so CompatHelper runs on whatever the latest formal release of Julia.